ipam/store.go

```diff
@@ -50,3 +51,47 @@ type IpamVm struct {
 	Id uint64 `sql:"AUTO_INCREMENT"`
 	//	IpamHostId sql.NullString
 }
+
+type ipamStore struct {
+	common.DbStore
+}
+
+func (ipamStore *ipamStore) addVm(stride uint, vm *Vm) error {
+	tx := ipamStore.DbStore.Db.Begin()
```

Comment:
Well....  I agree but the functionality does not exactly map well to RDBMS backin store and hiding it neatly doesn't work - yet.